packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -19,8 +18,9 @@ dependencies:
   flutter_local_notifications: ^13.0.0
   flutter_localizations:
     sdk: flutter
-  flutter_rx_bloc: ^5.0.1
+  flutter_rx_bloc: ^5.0.2
   flutter_secure_storage: ^7.0.1
+  go_router: 6.0.4
```

Comment:
Please do not use an exact version of the packages!

Reply:
Done